### PR TITLE
Improve mobile xp

### PR DESCRIPTION
**Pull request by AlexandreSi**
Make sure that every text field with `autofocus` props won't autofocus on mobile to prevent keyboard opening

### CODE REVIEW

File: newIDE/app/src/UI/SemiControlledTextField.js [hunk 1]
```diff
@@ -50,90 +45,94 @@ type Props = {|
...
+|};
+
 /**
  * This component works like a material-ui TextField, except that
  * the value passed as props is not forced into the text field when the user
  * is typing. This is useful if the parent component can do modifications on the value:
  * the user won't be interrupted or have the value changed until he blurs the field.
  */
-export default class SemiControlledTextField extends React.Component<
+const SemiControlledTextField = React.forwardRef<
```
Comment (ClementPasteau) [inline newIDE/app/src/UI/SemiControlledTextField.js:65]: 💌

File: newIDE/app/src/UI/SemiControlledTextField.js [hunk 2]
```diff
@@ -50,90 +45,94 @@ type Props = {|
...
-        type={type || 'text'}
-        ref={field => (this._field = field)}
-        value={this.state.focused ? this.state.text : value}
-        onFocus={event => {
-          this.setState({
-            focused: true,
-            text: this.props.value,
-          });
+  return (
+    // $FlowFixMe
```
Comment (ClementPasteau) [inline newIDE/app/src/UI/SemiControlledTextField.js:111]: is this flowfixme still required?

**Review by ClementPasteau — APPROVED** [review]
nice!

File: newIDE/app/src/UI/SemiControlledTextField.js (hunk 2 above)
Comment (AlexandreSi) [reply newIDE/app/src/UI/SemiControlledTextField.js:111]: Yes, I did not have the courage to fix it

**Pull request merged**